### DISCUSSION TITLE
Detectron2 model in Raspberry Pi4 8gb

### DISCUSSION THREAD

**Discussion by sourabmaity** [facebookresearch/detectron2#4499] [Q&A]
I created a custom weight using that weight in Raspberry Pi "CPU" I am getting 15 s delay for each frame ( from prediction to visualization) .
How to decrease this delay? or any Detectron2 nano model

Or How to use OpenCL in Detectron2?

**Comment by ExtReMLapin** [comment]
Detectron2 is cuda only AFAIK.
CPU prediction is always slow.

If I were you I would stick to the smallest possible architecture, check inference time on [zoo page](https://github.com/facebookresearch/detectron2/blob/main/MODEL_ZOO.md)


For embeded system, maybe take a look at this https://developer.nvidia.com/embedded/jetson-modules
